libmscore/style.cpp

```diff
@@ -61,13 +61,13 @@ struct StyleType {
 static const StyleType styleTypes[] {
       { Sid::pageWidth,               "pageWidth",               210.0/INCH },
       { Sid::pageHeight,              "pageHeight",              297.0/INCH }, // A4
-      { Sid::pagePrintableWidth,      "pagePrintableWidth",      190.0/INCH },
-      { Sid::pageEvenLeftMargin,      "pageEvenLeftMargin",      10.0/INCH  },
-      { Sid::pageOddLeftMargin,       "pageOddLeftMargin",       10.0/INCH  },
-      { Sid::pageEvenTopMargin,       "pageEvenTopMargin",       10.0/INCH  },
-      { Sid::pageEvenBottomMargin,    "pageEvenBottomMargin",    20.0/INCH  },
-      { Sid::pageOddTopMargin,        "pageOddTopMargin",        10.0/INCH  },
-      { Sid::pageOddBottomMargin,     "pageOddBottomMargin",     20.0/INCH  },
+      { Sid::pagePrintableWidth,      "pagePrintableWidth",      180.06/INCH },
```

Comment:
180.06 + 2 * 15.0 > 210.0, so that .06 should go I guess

Reply:
agreed, thx